Docker.DotNet.X509/RSAUtil.cs

```diff
@@ -19,20 +15,6 @@ public static X509Certificate2 GetCertFromPFX(string pfxFilePath, string passwor
             return new X509Certificate2(pfxFilePath, password);
         }
 
-#if (NET45 || NET46)
-        public static X509Certificate2 GetCertFromPFXSecure(string pfxFilePath, SecureString password)
```

Comment:
Yea its not X509Certificate2 its actually the .PrivateKey that I remember not being there. We need to support reading keys from .PEM files so we cant remove this